app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/SignInWithSAAccount.scala.html

```diff
@@ -0,0 +1,72 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.Layout
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.SignOutController
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.ReportSuspiciousIDController
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.AccountDetails
+
+
+@this(
+  layout: Layout,
+  govukSummaryList : GovukSummaryList,
+  govukButton: GovukButton
+)
+
+@(saAccountDetails: AccountDetails, signOutController:SignOutController, reportSuspiciousIDController: ReportSuspiciousIDController)(implicit request: Request[_], messages: Messages)
+
+@createSummaryListRow(key: String, value: String) =  {
+<div class="govuk-summary-list__row">
+ <dt class="govuk-summary-list__key">
+  @key
+ </dt>
+ <dd class="govuk-summary-list__value">
+  @value
+ </dd>
+</div>
+}
+
+@layout(pageTitle = Some(messages("signInAgain.title")), Some("?")) {
+
+ <h1 class="govuk-heading-xl">@messages("signInAgain.heading1")</h1>
+ <p class="govuk-body">@messages("signInAgain.paragraph")</p>
+ <h3 class="govuk-heading-m">@messages("signInAgain.heading2")</h3>
+ <dl class="govuk-summary-list">
+  @createSummaryListRow(messages("signInAgain.userId"), saAccountDetails.userId)
+  @if(saAccountDetails.email.isDefined) {
+  @createSummaryListRow(messages("signInAgain.email"), saAccountDetails.email.get)
+  }
+  @createSummaryListRow(messages("signInAgain.lastLoginDate"), saAccountDetails.lastLoginDate)
+  @for(mfaDetail <- saAccountDetails.mfaDetails) {
+  @createSummaryListRow(mfaDetail.factorName, mfaDetail.factorValue)
+  }
+ </dl>
+
+ <p class="govuk-body">
+   <a class="govuk-link" id="reportId" href="@reportSuspiciousIDController.view()">
```

Comment:
href = "@routes.ReportSuspiciousIDController.view" can be used just make sure your import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.routes